src/apps/testapps/testBBox.c

```diff
@@ -88,9 +88,16 @@ TEST(transmeridian) {
                         {-0.4, M_PI - 0.1}};
     const Geofence geofence = {.numVerts = 4, .verts = verts};
     const BBox expected = {0.4, -0.4, -M_PI + 0.1, M_PI - 0.1};
-    const GeoCoord inside = {-0.1, M_PI};
+    const GeoCoord insideOnMeridian = {-0.1, M_PI};
```

Comment:
IDK how you all feel about this idea, but I find it would be easier to read this code if it used [designated initializers](https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html):

```
const GeoCoord insideOnMeridian = { .lat = -0.1, .lon = M_PI }
```

Reply:
Yeah, we've been really inconsistent on this in tests, but I think that's probably the best/most legible option for most structs. `GeoCoord` might be the exception, because a) we instantiate a lot of them, and b) it's really equivalent to a `lat, lon` tuple, so instantiating it with the same syntax as a `double[]` array makes sense.

Reply:
I get that. Just a warning about designated initializers: it's not 100% portable, but should be okay on most platforms.